docs/installation-guide-mac.rst

```diff
@@ -0,0 +1,62 @@
+:orphan:
+
+Installation guide for macOS
+============================
+
+Follow these instructions to install the CityEnergyAnalyst (CEA) on a macOS system (tested with Mojave 10.14.5) from the source
+
+Prerequisites
+~~~~~~~~~~~~~
+
+* Download and install Git (64-bit).
+* Download and install Github Desktop (64-bit).

```

Comment:
This isn't strictly a pre-requisite. It's just nice-to-have if you're a beginner with git... for documenting the Installation, actual git commands are easier... There are also alternatives (`fork`, `gitk`, `gitkraken` etc.)

Reply:
In that case, that step should be deleted from all tutorials ([e.g.](https://city-energy-analyst.readthedocs.io/en/latest/installation-on-windows-manual.html))

Reply:
yeah. I know what you mean. hm.